automerge-backend-wasm/scripts/js_tests.sh

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+
+if [[ -e "../../automerge" ]]
```

Comment:
```suggestion
if [[ -d "../../automerge" ]]
```
I'd expect `automerge` to be a directory

Reply:
Done